e2e-tests/gatsby-image/package.json

```diff
@@ -7,12 +7,13 @@
     "cypress": "^3.1.0",
     "gatsby": "^2.0.118",
     "gatsby-image": "^2.0.29",
+    "gatsby-plugin-image": "static-image",
```

Comment:
Maybe these need to be use carets to make them upgrade-able?.. But this got things up and running for me 🤷‍♂️

Reply:
Once the new API is in master then we can move to a normal dependency